### ISSUE TITLE
No checksum provided for linux builds. Can't authorize as a result.

### ISSUE THREAD

**Issue by JNiena** [viktorgullmark/exilence-next#823]
**Describe the bug**

**To Reproduce**
1. Open the program.
2. Select your platform.
3. Click authorize.
4. After being redirected to the POE website, click authorize.

**Expected behavior**
The POE website should successfully authorize Exilence Next and you should be able to continue on with the application. What happens instead is that when you click the authorize button on the POE website, nothing will happen.

**Version**
v1.1.8 (Built on Linux)

**Additional context**
This is the error that is printed to the console: `Error: Cannot find latest-linux.yml in the latest release artifacts (https://github.com/viktorgullmark/exilence-next/releases/download/v1.1.8/latest-linux.yml): HttpError: 404`

There is a latest.yml file in the latest release, which seems to be a checksum for the windows version of the program, but that obviously won't work for the linux build.

It seems there is no checksum file "latest-linux.yml" which would be for linux builds, which is why that error is saying file doesn't exist when the GET request is sent.

**Comment by plastic-fork** [comment]
Use the release build instead, worked for me.

**Comment by JNiena** [comment]
That was the error that had occurred after I downloaded the source and tried to build it (all steps followed).

**Comment by plastic-fork** [comment]
> That was the error that had occurred after I downloaded the source and tried to build it (all steps followed).

Yeah i know but i meant the source code zip from the releases found here:
https://github.com/viktorgullmark/exilence-next/releases/tag/v1.1.8

I ran into the same issue as you, i tried the 1.1.8 source and it managed to build and run.
If you really want to run the latest git version you can do it with `npm start`
But as it stands you can't build and run the appimage unfortunately.
